### ISSUE TITLE
TGI Fridays - use UK-specific Wikidata entry?

### ISSUE THREAD

**Issue by rjw62** [osmlab/name-suggestion-index#10027]
The restaurant chain TGI Fridays has an NSI entry at https://nsi.guide/index.html?t=brands&k=amenity&v=restaurant&tt=fridays and uses brand:wikidata=[Q1524184](https://www.wikidata.org/wiki/Q1524184).

However, I've just noticed that one of the branches in the UK has been mapped in OSM with brand:wikidata=[Q112828145](https://www.wikidata.org/wiki/Q112828145) - a more specific entry for the UK branch of the chain.

Should we be using this more specific entry instead, and should NSI be updated to reflect this?

**Comment by bhousel** [comment]
It doesn't really make sense to me why there would be a different QID for "TGI Frdays in the UK".  and https://www.wikidata.org/wiki/Q112828145 looks pretty sparse.  Maybe it's a mistake?

**Comment by UKChris-osm** [comment]
I couldn't find the branch with the UK specific Wikidata, has it been changed?

I think it would be best to keep TGI Fridays as it is, without the UK specific Wikidata, as the brand is well mapped at the moment, and I don't see any particular benefit to the same brand having country specific Wikidata's unless there's a good reason.

**Comment by rjw62** [comment]
The argument for having a separate ID would be that the UK branches are run by a company that is a separate legal entity. I'm not sure of the precise setup, but I think it may be an independent business with a licensing deal, rather than just a subsidiary of the US parent company. (I've come across this as the UK company has recently gone into Administration and the sold to another firm.)

I altered the one instance of brand:wikidata=Q112828145 to Q1524184 for consistency with the others. The problem is that since the Wikidata item exists, it's not _wrong_ for UK-based TGI Fridays to be tagged with it, and arguably the more specific ID is more correct. This could lead to inconsistencies in the data, and iD will use the NSI to coerce editors to change the more specific ID back.

**Comment by rjw62** [comment]
I guess part of the problem here is that Wikidata often conflates the band and the company operating it. Ideally, I think there would be one Wikidata entry for "TGI Fridays" as the restaurant fascia brand, and other entries for the business entities that own/run the brand in different territories.

**Comment by 1ec5** [comment]
> Ideally, I think there would be one Wikidata entry for "TGI Fridays" as the restaurant fascia brand, and other entries for the business entities that own/run the brand in different territories.

That would be the most precise setup, and there is precedent for it, like Westfield (#2743) and [Shell](https://www.wikidata.org/wiki/User_talk:Mxn#On_the_topic_of_Shell), mainly because of piecemeal changes in ownership. Unfortunately, people regularly come along and incorrectly merge the brand-only item into the owning company’s item because they find it fussy or think it’s a mistake. It leads to plenty of confusion – see #9928 for one example. So we do attempt to distinguish the brand from its owner, expect to fight a bit of an uphill battle against drive-by simplifiers.

**Comment by UKChris-osm** [comment]
> The argument for having a separate ID would be that the UK branches are run by a company that is a separate legal entity. I'm not sure of the precise setup, but I think it may be an independent business with a licensing deal, rather than just a subsidiary of the US parent company. (I've come across this as the UK company has recently gone into Administration and the sold to another firm.)

True, but from my perspective, we'd be drilling down into to much detail when I see the NSI as simply a name / brand database, without having to focus on who runs this area, who operates those stores, etc. From a mapping perspective, all we're doing here is saying "this is a TGI Fridays", rather than trying to say "this is a TGI Friday's run by Thursdays Ltd".

It would be a lot easier if a brand QID existed separately from any organisation that owned / ran the business though, which is why I feel like an OSM / NSI based "_brand ID_" would be better than using wikidata, but that's a whole other discussion :)
